src/components/utils/index.js

```diff
@@ -130,14 +130,20 @@ export const DisplayComponentsUsingCss = styled(Components)`
         : ''}
 `
 
-export const formatUTC = (utcDate, utcOffset = 60, format = "D MMM 'YYYY") => {
+export const formatUTC = (
+  utcDate,
+  utcOffset = 60,
+  format = "D MMM 'YYYY",
+  day = 0
+) => {
   const targetTime = new Date(utcDate),
     now = new Date(),
     minutesToMilliseconds = 60000
   const localOffsetInMs = now.getTimezoneOffset() * minutesToMilliseconds
   const utcOffsetInMs = utcOffset * minutesToMilliseconds
+  const dayOffset = day * 1440 * minutesToMilliseconds
```

Comment:
@ric9176 what is 1440? I'd create a const for that number to give meaning, as we did with `minutesToMilliseconds = 60000`. Maybe `const dayOffset` is `const dayOffsetInMs`?